src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -944,7 +944,7 @@ static void WindowNewRidePaintRideInformation(
         money64 price = GetRideTypeDescriptor(item.Type).BuildCosts.TrackPrice;
         const auto& ted = GetTrackElementDescriptor(startPieceId);
         price *= ted.Price;
-        price = (price >> 17) * 10 * GetRideTypeDescriptor(item.Type).BuildCosts.PriceEstimateMultiplier;
+        price = ToGBP(price >> 17) * GetRideTypeDescriptor(item.Type).BuildCosts.PriceEstimateMultiplier;
```

Comment:
its debatable if this is really correct. The *10 here doesn't really mean convert from gbp to money32 its more of a fancy floor function okay it is kind of related but not really.